qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+the autogenerated clients for collections and points respectively. 
+
+The gRPC client uses type definitions for all the interactions with the server. They are autogenerated from the source code of Qdrant
+server, and we have to import them if we want to call any of the available methods.
+
+```python
+from qdrant_client import grpc
+```
+
+If your IDE does not support autocompletion for the autogenerated clients, you can refer to the documentation and [see the available
+fields for each of them](https://github.com/qdrant/qdrant/blob/master/docs/grpc/docs.md).
+
+#### Creating a collection
+
+Let's create a collection and add some points to it.
+
+```python
+response = await client.async_grpc_collections.Create(
+    grpc.CreateCollection(
+        collection_name="my_collection",
+        vectors_config=grpc.VectorsConfig(
+            params=grpc.VectorParams(
+                size=100,
+                distance=grpc.Distance.Cosine,
+            )
+        ),
+        quantization_config=grpc.QuantizationConfig(
+            scalar=grpc.ScalarQuantization(
+                type=grpc.QuantizationType.Int8,
+                always_ram=False,
+            )
+        ),
+        timeout=10,
+    )
+)
+```
+
+Our collection was set up for 100-dimensional vectors, cosine distance and 8-bit quantization. The timeout is set to 10 seconds, so
+if the server does not respond in this time, the request will be aborted.
+
+#### Checking collection info
+
+We can check the collection info to see if it was created successfully, and configured as we expected:
+
+```python
+response = await client.async_grpc_collections.Get(
+    grpc.GetCollectionInfoRequest(
+        collection_name="my_collection"
+    )
+)
+```
+
+The `response` object is going to be an instance of `grpc.GetCollectionInfoResponse` and we can access the fields we need or simply
+display its content.
+
+```python
+print(response)
+```
+
+```text
+result {
+  status: Green
+  optimizer_status {
+    ok: true
+  }
+  segments_count: 8
+  config {
+    params {
+      shard_number: 1
+      on_disk_payload: true
+      vectors_config {
+        params {
+          size: 100
+          distance: Cosine
+        }
+      }
+      replication_factor: 1
+      write_consistency_factor: 1
+    }
+    hnsw_config {
+      m: 16
+      ef_construct: 100
+      full_scan_threshold: 10000
+      max_indexing_threads: 0
+      on_disk: false
+    }
+    optimizer_config {
+      deleted_threshold: 0.2
+      vacuum_min_vector_number: 1000
+      default_segment_number: 0
+      indexing_threshold: 20000
+      flush_interval_sec: 5
+      max_optimization_threads: 1
+    }
+    wal_config {
+      wal_capacity_mb: 32
+      wal_segments_ahead: 0
+    }
+    quantization_config {
+      scalar {
+        type: Int8
+        always_ram: false
+      }
+    }
+  }
+  indexed_vectors_count: 0
+}
+time: 1.3022e-05
+```
+
+As you can see, the collection was created properly and is ready to be used.
```

Comment:
Where can I go more to learn about what all these fields mean? Is there a glossary or page that explains all this?

Reply:
OK, can you link to that please with something about "to understand more of the output"....